src/components/Sections/SectionNumber.js

```diff
@@ -58,7 +60,8 @@ const SectionNumber = ({ data, layout, style, sign, signAlignment, title, titleS
           style={{ color }}
         >
           {signAlignment === 'left' && signElement}
-          {numberToShortString(curr)}
+          {formatNumberValue(curr,
+            valuesFormat === WIDGET_VALUES_FORMAT.percentage ? WIDGET_VALUES_FORMAT.regular : valuesFormat)}
```

Comment:
Why do we need to change the format when it's "percentage" ?
If we have some special requirement for percentage can it be implemented inside "formatNumberValue"?

Reply:
number widget already supports generic suffix.
negative, format-number-value is a general util that serves others and it should remain "caller agnostic"